wikipron/languagecodes.py

```diff
@@ -252,10 +252,89 @@
     "tetelcingo nahuatl": "Tetelcingo Nahuatl",
     # Bouyei. ISO 639-3 only.
     "pcc": "Bouyei",
+    "bouyei": "Bouyei",
     # Lamboya. ISO 639-3 only.
-    "lmy": "Lamboya",
+    "lmy": "Laboya",
+    "laboya": "Laboya",
     # Moroccan Arabic. ISO 639-3 only.
     "ary": "Moroccan Arabic",
+    "moroccan arabic": "Moroccan Arabic",
     # Mandarin Chinese. ISO 639-3 only.
-    "cmn": "Chinese"
+    "cmn": "Chinese",
+    "chinese": "Chinese",
+    # Abkhaz. Would be Abkhazian in ISO 639.
+    "abk": "Abkhaz",
+    "abkhaz": "Abkhaz",
+    # Avar. Would be Avaric in ISO 639.
+    "ava": "Avar",
+    "avar": "Avar",
+    # Buryat. Would be Buriat in ISO 639.
+    "bua": "Buryat",
+    "buryat": "Buryat",
+    # Chukchi. Would be Chukot in ISO 639.
+    "ckt": "Chukchi",
+    "chukchi": "Chukchi",
+    # Burushaski. ISO 639-3 only.
+    "bsk": "Burushaski",
+    "burushaski": "Burushaski",
+    # Evenki. Not in iso639 lib.
+    "evn": "Evenki",
+    "evenki": "Evenki",
+    # Southern Yukaghir. Not in iso639 lib.
+    "yux": "Southern Yukaghir",
+    "southern yukaghir": "Southern Yukaghir",
+    # Tundra Nenets. Not in iso639 lib.
+    "yrk": "Tundra Nenets",
+    "tundra nenets": "Tundra Nenets",
+    # Estonian. Not in iso639 lib.
+    "ekk": "Estonian",
+    "estonian": "Estonian",
+    # Livvi. Not in iso639 lib.
+    "olo": "Livvi",
+    "livvi": "Livvi",
+    # Kildin Sami. Not in iso639 lib.
+    "sjd": "Kildin Sami",
+    "kildin sami": "Kildin Sami",
+    # Northern Yukaghir. Not in iso639 lib.
+    "ykg": "Northern Yukaghir",
+    "northern yukaghir": "Northern Yukaghir",
+    # Nanai. Not in iso639 lib.
+    "gld": "Nanai",
+    "nanai": "Nanai",
+    # Persian. Not in iso639 lib.
+    "pes": "Persian",
+    "persian": "Persian",
+    # Greenlandic. Would be Kalaallisut in ISO 639.
+    "kal": "Greenlandic",
+    "greenlandic": "Greenlandic",
+    # Khanty. Not in iso639 lib.
+    "kca": "Khanty",
+    "khanty": "Khanty",
+    # Ket. Not in iso639 lib.
+    "ket": "Ket",
+    "ket": "Ket",
```

Comment:
I suppose when the name and the code are the same we can break from procedure and leave out the lowercase Wiktionary language name.

Reply:
It makes sense, I didn't pay attention to this when duplicating the lines.